### DISCUSSION TITLE
Jamulus 3.8 Feedback detected when there isn't any

### DISCUSSION THREAD

**Discussion by bonfigleo** [jamulussoftware/jamulus#1823] [Technical Support & Help]
Hello,

I just updated to 3.8 and I am experiencing false detections of feedback.  This occurs when there is an extremely loud noise.

I can replicate by connecting to my server while nobody else is connected and muting my channel.  If I shout into the mic, I will get the feedback detected dialogue and mute myself will get checked. 

I would, at least like a way to prevent the "Mute Myself" from getting checked.

Thanks!

**Comment by henkdegroot** [comment]
> I would, at least like a way to prevent the "Mute Myself" from getting checked.

I would say if you want that, you should disable the feedback detection option....as that is what the option will do. You can find the option in the Settings dialog -> Advanced Setting tab.

**Reply by bonfigleo** [reply]
I thought that too.  Unfortunately, that setting does not appear to do anything.

**Reply by hoffie** [reply]
> I thought that too. Unfortunately, that setting does not appear to do anything.

So, are you saying that disabling the feedback detection does... not disable the feedback detection? This would certainly be a bug which should be fixed.

From a quick look at the code, the disabling logic looks sane -- even when the timer has already been started.

cc @JohannesBrx

**Comment by ann0see** [comment]
There’s also a checkbox in the error message coming up once it kicks in. Can you try this one?
